app/javascript/app/components/home/flagship-programmes/flagship-programmes-styles.scss

```diff
@@ -0,0 +1,47 @@
+@import '~styles/settings.scss';
+@import '~styles/layout.scss';
+
+.container {
+  @include row();
+
+  position: relative;
+  height: 700px;
+
+  @media #{$tablet-portrait} {
+    height: 1080px;
+  }
+
+  @media #{$tablet-landscape} {
+    height: 600px;
+  }
+}
+
+.title {
+  font-family: Lato;
+  font-size: $font-size-large;
+  color: $theme-color;
+  margin-bottom: $gutter-padding;
+}
+
+.grid {
+  height: 90%; // avoid the children going outside of div
```

Comment:
I don't understand it very well, is is necessary for the child elements of the grid? seems to have a weird behaviour in the responsive being outside so the footer doesn't appear correct.
![image](https://user-images.githubusercontent.com/10500650/44196157-73a2ed80-a13b-11e8-9f91-e4578cf08d27.png)